extensions/rich_text_components/Video/Video.html

```diff
@@ -1,7 +1,8 @@
 <script type="text/ng-template" id="richTextComponent/Video">
   <center>
-    <iframe width="500" height="280"
-            ng-src="<[videoUrl]>" frameborder="0" allowfullscreen tabindex="<[tabIndexVal]>">
-    </iframe>
+    <div style="position: relative; padding-bottom: 56.25%; padding-top: 25px; height: 0;">
```

Comment:
Why is the padding-top a px number, and the padding-bottom a percentage? Also, what is the purpose of introducing padding here?

Is it intentional to set the height to 0? It seems a bit strange because it contains a video of non-zero height.